webapi/webapi-devicecapabilities-w3c-tests/w3c-devicecapabilities-app/testscripts/DisplayUnit.feature

```diff
@@ -0,0 +1,18 @@
+Feature: devicecapabilities
+ Scenario: DisplayUnit
+  When launch "w3c-devicecapabilities-app"
+   And I go to "devicecapabilities/DisplayUnit-manual.html"
+   And I wait 4 seconds
+  Then I should not see "Failed"
+  Then I should see "DisplayUnit id"
+  Then I should see "DisplayUnit name"
+  Then I should see "DisplayUnit primary"
+  Then I should see "DisplayUnit external"
+  Then I should see "DisplayUnit deviceXDPI"
```

Comment:
What about the value of these attributes? You can refer to this test https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/usecase/usecase-webapi-xwalk-tests/testscripts/DeviceCapabilities.feature